bigbluebutton-html5/imports/ui/components/shortcut/component.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+
+export default class Shortcut extends Component {
+  constructor() {
+    super();
+
+    this.handleShortcut = this.handleShortcut.bind(this);
+  }
+
+  componentDidMount() {
+    document.addEventListener('keydown', this.handleShortcut, false);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.handleShortcut, false);
+  }
+
+  handleShortcut(event) {
+    const { keyCombo } = this.props;
+
+    const combo = keyCombo.toUpperCase();
+    const keys = combo.split('+');
+
+    if (event.key.toUpperCase() !== keys[keys.length - 1]) return;
+    if (!event.ctrlKey && (combo.includes('CONTROL') || combo.includes('CTRL'))) return;
+    if (!event.altKey && (combo.includes('ALT') || combo.includes('OPTION'))) return;
+    if (!event.shiftKey && keys.includes('SHIFT')) return;
+
+    this.element.firstChild.click();
+  }
+
+  render() {
+    return (
+      <span ref={(node) => { this.element = node; }}>
```

Comment:
Can`t you just clone the children and add the `ref`? I can see this span breaking layout stuff pretty easy.